src/js/app.js

```diff
@@ -29,8 +29,20 @@ const app = {
         $('#todos').append($todoContainer);
         
         app.counter = app.counter + 1;
+
+        $todoCheckbox.change(function () {
+        
+            if(this.checked == true){
```

Comment:
Acuérdate que en el caso de las condiciones, se considera una buena práctica usar `===` (igualdad estricta) para asegurarnos que no tengamos problemas de [`type coercion`](https://stackoverflow.com/questions/19915688/what-exactly-is-type-coercion-in-javascript).